torch/_dynamo/variables/builder.py

```diff
@@ -320,6 +320,10 @@ def _wrap(self, value):
         if id_dispatch is not None:
             return id_dispatch(self, value)
 
+        # Note - There are some nested values where types mismatch!
+        # We want to get those out and wrap those.
+        value = inspect.getattr_static(value, "_torchdynamo_inline", value)
```

Comment:
Don't you want to do this before the type()/id() matches then?

Reply:
I was not sure, actually. Reading the code - it does not matter today, but maybe it will? I'll move it up.

Reply:
Actually, I think here is better, this can only happen on types that are not subject to the above.